clarity_ext/units.py

```diff
@@ -0,0 +1,30 @@
+import math
+import logging
+
```

Comment:
I think it might be valid to have it in a package called e.g. `utils` or `misc` - but I think the modules here are quickly getting really large, e.g. the domain module. That's very common in Python projects, but personally I think it's a bad habit, since it makes getting an overview of the project difficult.

Taking a successful Python library as an example, Luigi (https://github.com/spotify/luigi/tree/master/luigi), they often have small modules. They then import these modules to the package level by importing them in `__init__.py`. See [here](https://github.com/spotify/luigi/blob/master/luigi/task_status.py) and [here](https://github.com/spotify/luigi/blob/master/luigi/__init__.py).

In this case, I believe that this class will get ~1 page long when finished, and it contains a clearly defined thing (something that handles unit conversion), so I think it should be in its own module. I'm not sure I see the advantage of moving it into another package though - since it's unclear why this is `misc`, but not for example `pdf.py`.